testing/sequoia-sqv/APKBUILD

```diff
@@ -31,7 +31,10 @@ _cargo_env() {
 build() {
 	_cargo_env
 
-	sed -i -e '/^version =/s/=.*$/= "'"$pkgver"'"/' sqv/Cargo.toml
+	sed -i.orig -e '/^version =/s/=.*$/= "'"$pkgver"'"/' sqv/Cargo.toml
```

Comment:
Could you add a comment explaining the goal of this? Thanks.

Reply:
Previous source archives did not update the version to match. This change and diff is for detecting when that happened again.